### PR TITLE
Fix some inconsistencies in network policy documentation.

### LINKED ISSUE #5393: document egress networkpolicy support
**Problem:**
EgressRule has been added to Network Policy but no docs exist 

**Proposed Solution:**
Examine the code comments for the [Egress PR](https://github.com/kubernetes/kubernetes/pull/51351) and add the necessary examples to the network-policy documentation. Special attention needs to be placed on how to create a default policy for Egress vs. Ingress. 

**Page to Update:**
https://kubernetes.io/docs/concepts/services-networking/network-policies/

Kubernetes Version: 1.8

### PR DESCRIPTION
**Pull request by mindprince**
This fixes #5393.

This change is [Reviewable](https://reviewable.kubernetes.io/reviews/kubernetes/kubernetes.github.io/5915)

### CODE REVIEW

File: docs/concepts/services-networking/network-policies.md [hunk 1]
```diff
@@ -72,25 +72,21 @@ __Mandatory Fields__: As with all other Kubernetes config, a `NetworkPolicy` nee
...
 
 __policyTypes__: Each `NetworkPolicy` includes a `policyTypes` list which may include either `Ingress`, `Egress`, or both. The `policyTypes` field indicates whether or not the given policy applies to ingress traffic to selected pod, egress traffic from selected pods, or both. If no `policyTypes` are specified on a NetworkPolicy then by default `Ingress` will always be set and `Egress` will be set if the NetworkPolicy has any egress rules.
 
-__ingress__: Each `NetworkPolicy` may include a list of whitelist `ingress` rules.  Each rule allows traffic which matches both the `from` and `ports` sections. The example policy contains a single rule, which matches traffic on a single port, from either of two sources, the first specified via a `namespaceSelector` and the second specified via a `podSelector`.
+__ingress__: Each `NetworkPolicy` may include a list of whitelist `ingress` rules.  Each rule allows traffic which matches both the `from` and `ports` sections. The example policy contains a single rule, which matches traffic on a single port, from either of three sources, the first specified via an `ipBlock`, the second via a `namespaceSelector` and the third via a `podSelector`.
```
Comment (xiangpengzhao) [inline docs/concepts/services-networking/network-policies.md]: I'm not a native speaker. Is `either of three` correct?

Comment (mindprince) [reply docs/concepts/services-networking/network-policies.md]: Updated to say `one of three`.

**Comment by xiangpengzhao** [comment]
@mindprince thanks! Merge per Tim's comment.

**Pull request merged**